lib/helper/Appium.js

```diff
@@ -189,10 +189,7 @@ class Appium extends WebdriverIO {
   _after() {
     if (this.options.restart) return this.browser.end();
     if (this.isWeb && !this.platform) {
-      this.debugSection('Session', 'cleaning cookies and localStorage');
```

Comment:
@DavertMik yes, I created this branch from `appium-web-tests-improve` branch to have opportunity to test it locally with Appium helper.